assets/javascript/game.js

```diff
@@ -0,0 +1,89 @@
+
+var tpWords = ["bob", "giant", "cooper", "diane", "cave", "owl", "coffee"];
+let numGuesses = 0; // This stores the current amount of guessed letters
+let rand = Math.floor(Math.random() * tpWords.length);
+let word = tpWords[rand]; //This is the word chosen at random
+let splitWord = word.split('');
+let nameLength = word.length;
+let correctNumSpace = [];
+let hiddenAnswer = word;
+let correctKeys = [];
+let wrongKeys = [];
+
+// Everything about this  line is a variable ********************************************
+
+// This add an _ to each letter in chosen word
+for (var n = 0; n < nameLength; n++) {
```

Comment:
most of the time you can make all for loops for(var i = 0 etc.
the only time j, k, l, etc are needed are in nested loops e.g.:
```javascript
for (var i = 0; i < arr1.length; i++) {
  for (var j = 0; j < arr2.length; j++) {
     //code here
  }
}
```